tasks.py

```diff
@@ -196,7 +196,27 @@ def translate(c):
     manage(c, "compilemessages")
 
 
-@task(post=[rebuild_models, rebuild_thumbnails])
+@task
+def backup(c):
+    """Backup the database and media files."""
+
+    print("Backing up InvenTree database...")
+    manage(c, "dbbackup --noinput --clean --compress")
+    print("Backing up InvenTree media files...")
+    manage(c, "mediabackup --noinput --clean --compress")
+
+
+@task
+def restore(c):
```

Comment:
Can we allow the user to specify *which* backup to restore from?

Reply:
Yes, that is theoretically possible; Current method is to use the latest